components/Chips/tests/unit/ChipViewFontThemerTests.m

```diff
@@ -0,0 +1,100 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialThemes.h"
+#import "MaterialChips.h"
+#import "MDCChipViewFontThemer.h"
+
+@interface FakeFontScheme : NSObject <MDCFontScheme>
```

Comment:
you could use our basic font scheme and set your own values instead of this.

Reply:
Done